### PR TITLE
tests: Fix linearizability nightly

### PR DESCRIPTION
**Pull request by serathius**

### CODE REVIEW

File: .github/workflows/linearizability-nightly.yaml [hunk 1]
```diff
@@ -1,9 +1,12 @@
 name: Linearizability Nightly
+permissions: read-all
 on:
   # schedules always run against the main branch, hence we have to create separate jobs
   # with individual checkout actions for each of the active release branches
   schedule:
     - cron: '25 9 * * *' # runs every day at 09:25 UTC
+  push: {}
+  pull_request: {}
```
Comment (ahrtr) [inline .github/workflows/linearizability-nightly.yaml:9]: not sure why adding this? `on.schedule` already defines the time to run the workflow, usually we don't need `on.push` or `on.pull_request` in this case.

Comment (serathius) [reply .github/workflows/linearizability-nightly.yaml:9]: No this is temporary to make sure that the test pass.

**Review by ahrtr — APPROVED** [review]
Note: the Linearizability test took more than 70 minutes. 

@github-actions
Linearizability Nightly / test-main / test (pull_request) Successful in 76m
[Details](https://github.com/etcd-io/etcd/actions/runs/3962607183/jobs/6789477268)
@github-actions
Linearizability Nightly / test-35 / test (pull_request) Successful in 78m
[Details](https://github.com/etcd-io/etcd/actions/runs/3962607183/jobs/6789477461)
@github-actions
Linearizability Nightly / test-34 / test (pull_request) Successful in 82m

**Comment by serathius** [comment]
Removed presubmit runs to prepare for merge.

**Pull request merged**